code/modules/martial_arts/martial.dm

```diff
@@ -195,12 +195,17 @@
 	icon = 'icons/obj/library.dmi'
 	icon_state = "cqcmanual"
 
+/obj/item/CQC_manual/chef
+	desc = "A small, black manual. Written on the back it says: Bringing the home advantage with you."
+
 /obj/item/CQC_manual/attack_self(mob/living/carbon/human/user)
 	if(!istype(user) || !user)
 		return
 	to_chat(user, "<span class='boldannounce'>You remember the basics of CQC.</span>")
 
 	var/datum/martial_art/cqc/CQC = new(null)
+	var/datum/martial_art/cqc/under_siege/CCQC = new(null)
```

Comment:
Making a new one here is a bit odd. Best to include the removing of the current martial arts thing in the `teach` proc which overrides the old one

Reply:
After doing some testing it seems that removing the old CQC isn't even needed at all